watchdog/src/main.rs

```diff
@@ -0,0 +1,5 @@
+// Makes exported items available to the crate root.
```

Comment:
nit: I'd recommend using the same pattern as we do in the bitcoin canister. There, we define the endpoints in `main.rs`, not in `lib.rs`.

Reply:
I started with this approach, but the problem is that functions declared in `main.rs` are not visible in the underlying crates, which is needed to register transformation functions. That's why we have a compromise here: all endpoints are declared in `lib.rs`, which makes transformation functions visible downstream and makes them available to register the requests.